Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Dashboard/Ride.hs

```diff
@@ -52,6 +53,9 @@ mkCommonBookingLocation BookingLocation {..} =
       ..
     }
 
+mkCommonEstimateDistance :: HighPrecMeters -> Common.HighPrecMeters
+mkCommonEstimateDistance HighPrecMeters {..} = Common.HighPrecMeters {..}
```

Comment:
Isn't `HighPrecMeters` and `Common.HighPrecMeters` the same?
If not, why do we need two `HighPrecMeters` types?

Reply:
they are same , but we need to cast it because we are using it in the dashboard side.

Reply:
Why to use different type in dashboard @prashant601 ? We can just use the same Kernel type.
Type case is mostly for the types which are defined in either bap and bpp but for the types defined in Kernel, we don't need to do type cast. We can just import them from kernel.

Reply:
Okay, got it. 
I have made the changes accordingly.